HHS/Head-Start-TTADP
# Edit CDI grant assignments

## Description of change

* Allow for CDI grants to be assigned to the correct region and grantee (https://ocio-jira.acf.hhs.gov/browse/TTAHUB-93)

## How to test 93

1. Log into staging as an admin
2. Click admin -> CDI
3. Assign a CDI grant to a grantee
4. Ensure that grantee now appears in the correct region with that grant number


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
